{
  "uid" : "b71833c3b475e41",
  "name" : "test_02_file[file_name_info2]",
  "fullName" : "TestCase.test_vue.Test_Login_Vue#test_02_file",
  "historyId" : "c5a62fc1f8d1828e03df3941bf28128b",
  "time" : {
    "start" : 1647333018279,
    "stop" : 1647333018279,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "ValueError: too many values to unpack (expected 5)",
  "statusTrace" : "self = <TestCase.test_vue.Test_Login_Vue object at 0x7ffc42ab26a0>\nfile_name_info = [None, None, None, None, None, None]\n\n    @pytest.mark.parametrize(\"file_name_info\", read_excel(\"./Data/data.xlsx\", \"export\"))\n    def test_02_file(self, file_name_info):\n        # print(file_name_info)\n>       index, case_name, file_name, is_exe, result = file_name_info\nE       ValueError: too many values to unpack (expected 5)\n\nTestCase/test_vue.py:33: ValueError",
  "flaky" : false,
  "newFailed" : false,
  "newBroken" : false,
  "newPassed" : false,
  "retriesCount" : 7,
  "retriesStatusChange" : false,
  "beforeStages" : [ {
    "name" : "_Class__pytest_setup_class",
    "time" : {
      "start" : 1647333013474,
      "stop" : 1647333015238,
      "duration" : 1764
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "afterStages" : [ {
    "name" : "_Class__pytest_setup_class::0",
    "time" : {
      "start" : 1647333018281,
      "stop" : 1647333020406,
      "duration" : 2125
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "parentSuite",
    "value" : "TestCase"
  }, {
    "name" : "suite",
    "value" : "test_vue"
  }, {
    "name" : "subSuite",
    "value" : "Test_Login_Vue"
  }, {
    "name" : "host",
    "value" : "localhost"
  }, {
    "name" : "thread",
    "value" : "10936-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "TestCase.test_vue"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "file_name_info",
    "value" : "[None, None, None, None, None, None]"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "6a65faeeeb28cf49",
      "status" : "broken",
      "statusDetails" : "ValueError: too many values to unpack (expected 5)",
      "time" : {
        "start" : 1647332889475,
        "stop" : 1647332889476,
        "duration" : 1
      }
    }, {
      "uid" : "52e6aa58a8b55e4a",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: Can not connect to the Service chromedriver",
      "time" : {
        "start" : 1647331833458,
        "stop" : 1647331833458,
        "duration" : 0
      }
    }, {
      "uid" : "5a748684daa2c25c",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: Can not connect to the Service chromedriver",
      "time" : {
        "start" : 1647331748457,
        "stop" : 1647331748457,
        "duration" : 0
      }
    }, {
      "uid" : "e9b0e8d6621c2214",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: Can not connect to the Service chromedriver",
      "time" : {
        "start" : 1647331604446,
        "stop" : 1647331604446,
        "duration" : 0
      }
    }, {
      "uid" : "21e543c87ce21fd5",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: Can not connect to the Service chromedriver",
      "time" : {
        "start" : 1647331512620,
        "stop" : 1647331512620,
        "duration" : 0
      }
    }, {
      "uid" : "c6c007719b015716",
      "status" : "broken",
      "statusDetails" : "ValueError: too many values to unpack (expected 5)",
      "time" : {
        "start" : 1647309470973,
        "stop" : 1647309470973,
        "duration" : 0
      }
    }, {
      "uid" : "4798a397e8052fbc",
      "status" : "broken",
      "statusDetails" : "ValueError: too many values to unpack (expected 5)",
      "time" : {
        "start" : 1647309360709,
        "stop" : 1647309360709,
        "duration" : 0
      }
    } ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "b71833c3b475e41.json",
  "parameterValues" : [ "[None, None, None, None, None, None]" ]
}